bindata/etcd/etcd-member-remove.sh

```diff
@@ -8,30 +12,31 @@ if [[ $EUID -ne 0 ]]; then
   exit 1
 fi
 
-usage () {
+function usage {
     echo 'The name of the etcd member to remove is required: ./etcd-member-remove.sh $etcd_name'
     exit 1
 }
 
+# download etcdctl from upstream release assets
+function dl_etcdctl {
+  local etcdimg="registry.svc.ci.openshift.org/ci-op-l7y8kffx/stable@sha256:c576980bdce812e5d000e6f96fc238d75688da01f767cdaed6cbf04c40c600fa"
+  local etcdctr=$(podman create ${etcdimg})
+  local etcdmnt=$(podman mount "${etcdctr}")
+  cp ${etcdmnt}/bin/etcdctl $ASSET_DIR/bin
+  umount "${etcdmnt}"
+  podman rm "${etcdctr}"
+  $ETCDCTL_BIN version
+}
+
+### main 
 if [ "$1" == "" ]; then
     usage
 fi
 
-ETCD_NAME=$1
-ASSET_DIR=/home/core/assets
-ASSET_DIR_TMP="$ASSET_DIR/tmp"
-ETCDCTL=$ASSET_DIR/bin/etcdctl
-ETCD_DATA_DIR=/var/lib/etcd
-CONFIG_FILE_DIR=/etc/kubernetes
-
-source "/usr/local/bin/openshift-recovery-tools"
-
-function run {
-  init
-  dl_etcdctl
-  backup_etcd_client_certs
-  etcd_member_remove $ETCD_NAME
-}
+NAME="$1"
 
-run
+source etcd.env
 
+dl_etcdctl
+ID=$($ETCDCTL_WITH_TLS member list | awk -F "," "/\s${NAME}\,/"'{print $1}')
+etcdctl member remove $ID
```

Comment:
Just make them pass the ID directly.  Any reason not to?

Reply:
Because it is easier to pass the name. When they use the name, they are sure about what they are removing. The probability of removing a wrong member is less likely when names are used.

Reply:
> Because it is easier to pass the name. The assumption is that the probability of removing a wrong member is less likely when names are used.

given bash complexity against human readable output, I'm against but will defer to @hexfusion

Reply:
I really like the usability can we simplify bash as compromise
 
`etcdctl member list | awk -F, "/$NAME/{print \$1}"`

Reply:
That is not exact match. If someone give infra1, it matches infra10, infra11, infra12 etc.

Reply:
doh :)

`etcdctl member list | awk -F, "\$0 ~ /\s${NAME},/{print \$1}"`

Reply:
I guess the point is we need this to be simpler or pass the ID.